### DISCUSSION TITLE
Flags for languages

### DISCUSSION THREAD

**Discussion by mattheuswhale** [adityatelange/hugo-PaperMod#1480] [General]
How do I use flags to represent languages? I thought this was the default behaviour, but apparently something changed somewhere in the hugo updates. The papermod example website seems to have French as an icon so I'm wondering how I can do the same. 
    
I've tried setting the languageName and languageCode but it didn't make any changes to my website. Currently it only displays language code(ex: En in place of English) which makes it hard to read.

**Comment by jakubmeysner** [comment]
You need to set `params.displayFullLangName` to `true`. After that, `languageName` will be used, inside of which you can use [emojis](https://gohugo.io/quick-reference/emojis/#flags).[^1]

[^1]: https://github.com/adityatelange/hugo-PaperMod/blob/master/layouts/partials/translation_list.html
